### PR TITLE
updating to 01.00.29b1

### PR DESCRIPTION
**Pull request by mauzey1**

### CODE REVIEW

File: Tables/CMIP6_Efx.json [hunk 1]
```diff
@@ -54,7 +54,7 @@
             "frequency": "fx", 
             "modeling_realm": "land", 
             "standard_name": "soil_hydraulic_conductivity_at_saturation", 
-            "units": "1e-6 m s-1", 
+            "units": "micron s-1",
```
Comment (doutriaux1) [inline Tables/CMIP6_Efx.json]: I checked `micron` IS in fact in udunits

File: Tables/CMIP6_Emon.json [hunk 2]
```diff
@@ -2232,10 +2232,10 @@
             "frequency": "mon", 
             "modeling_realm": "atmos", 
             "standard_name": "eastward_atmosphere_dry_static_energy_transport_across_unit_distance", 
-            "units": "1.e6 J m-1 s-1", 
+            "units": "MJ m-1 s-1",
```
Comment (doutriaux1) [inline Tables/CMIP6_Emon.json]: Are we positive `udunits` can handle `MJ`?

File: Tables/CMIP6_coordinate.json [hunk 3]
```diff
@@ -437,20 +437,20 @@
             "long_name": "depth", 
             "climatology": "", 
             "formula": "", 
-            "must_have_bounds": "no", 
+            "must_have_bounds": "yes",
```
Comment (doutriaux1) [inline Tables/CMIP6_coordinate.json]: @taylor13 please double check this

File: Tables/CMIP6_Emon.json (hunk 2 above)
Comment (taylor13) [reply Tables/CMIP6_Emon.json]: It should be able to according to the documentation.

**Pull request closed without merging**